pkgs/flasksample/default.nix

```diff
@@ -1,16 +1,23 @@
-{ lib, python3Packages, fetchFromGitHub }:
-
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+let
+  src = fetchFromGitHub {
```

Comment:
I would have expected to be able to provide an argument for a path to pull via the fetch.

I read the docs and found that `fetchFromGitHub` pulls a compressed version of the code at the specified sha.  If you specify to pull submodules, `fetchgit` is used.  `fetchgit` also does not provide an argument to pull a path from a specified repo.  :cry:

Reply:
I thought the same thing!

Ended up leverage this example from azure-cli: https://github.com/NixOS/nixpkgs/blob/c5911744de78e8dd104320fde177dfe2b7b49d85/pkgs/tools/admin/azure-cli/default.nix#L20